### PR TITLE
update example for `notReverse`, in Course.List

### PR DESCRIPTION
**Pull request by toh995**

### CODE REVIEW

File: src/Course/List.hs [hunk 1]
```diff
@@ -323,7 +323,7 @@ produce f x =
 -- >>> notReverse Nil
 -- []
 --
--- prop> \x y -> let types = x :: List Int in notReverse x ++ notReverse y == notReverse (y ++ x)
+-- prop> \x y -> let types = x :: List Int in notReverse x ++ notReverse y == notReverse (x ++ y)
```
Comment (toh995) [inline src/Course/List.hs:326]: This seems more correct to me (please let me know if I'm off-base here!)

Comment (tonymorris) [reply src/Course/List.hs:326]: You are mistaken. Try it!

Comment (toh995) [reply src/Course/List.hs:326]: Ah - I was under the impression that the solution would look like:
```.hs
notReverse :: List a -> List a
notReverse = id
```

But after looking at the solution, I see my mistake. Thanks!

Comment (toh995) [reply src/Course/List.hs:326]: (Should have looked at the solutions first... lol)

**Pull request closed without merging**